hack/verify-shellcheck.sh

```diff
@@ -54,7 +54,7 @@ if [ ! -f "$SHELLCHECK" ]; then
   # install buildifier
   cd "${TMP_DIR}" || exit
   DOWNLOAD_FILE="${VERSION}.${OS}.x86_64.tar.xz"
-  curl -L https://storage.googleapis.com/shellcheck/"${DOWNLOAD_FILE}" -o "${TMP_DIR}/shellcheck.tar.xz"
+  curl -L https://github.com/koalaman/shellcheck/releases/download/v0.7.0/"${DOWNLOAD_FILE}" -o "${TMP_DIR}/shellcheck.tar.xz"
```

Comment:
Can you adjust the script's `VERSION` var to be `v0.7.0` and then update this URL to use the var in the 2 places needed? (i.e. match the placeholders in the URL in https://github.com/koalaman/shellcheck/issues/1871)